docs/conf.py

```diff
@@ -146,6 +146,7 @@
 gen_package(root, "wpimath.estimator")
 gen_package(root, "wpimath.filter")
 gen_package(root, "wpimath.geometry")
+gen_package(root, "wpimath.interpolation")
```

Comment:
Forgot to include this in api.rst; everything else looks fine, merge at will once this is green